src/models/mod.rs

```diff
@@ -84,7 +87,7 @@ pub(crate) fn extract_action_type_and_value_from_action_view(
             ActionKind::AddKey,
             json!({
                 "public_key": public_key,
-                "access_key": access_key,
+                "access_key": AccessKeyView::from(access_key),
```

Comment:
This whole `match` in `extract_action_type_and_value_from_action_view` seems to belong to `serializers`. Yet, I think we can get away with it as is for now. I only suggest you use `AccessKeyView` by its path as the name is colliding with the nearcore struct name:

```suggestion
                "access_key": self::serializers::AccessKeyView::from(access_key),
```

P.S. I suggest we call the module with `s` at the end (`serializers`).